tools/submission/submission-checker.py

```diff
@@ -549,6 +549,10 @@
     "offline": "Offline",
 }
 
+INFERRED_RESULTS_VARIABLES = {
+    "singlestream_to_multistream_factor": 8
```

Comment:
I think the factor of 8 should come from `samples_per_query`. Can you please check it is set to 8 for MultiStream?

Reply:
`samples_per_query` is actually 1. It is taken from the logs, and since it is running SingleStream it becomes 1. We could change it in the case we are inferring MultiStream